benches/cctv.rs

```diff
@@ -0,0 +1,89 @@
+use std::io::BufRead;
+
+use criterion::{criterion_group, criterion_main, BatchSize, Criterion};
+use crypto_bigint::U1024;
+use rand_chacha::ChaCha8Rng;
+
+use crypto_primes::{is_prime_with_rng, sieve_and_find, SieveFactory};
+use rand_core::SeedableRng;
+
+struct HardcodedSieve {
+    candidates: Vec<String>,
+    idx: usize,
+}
+impl HardcodedSieve {
```

Comment:
Can you put a blank line between top level items here? `rustfmt` doesn't do it for some reason, and I don't think there's a corresponding setting.